Sources/App/Models/Package.swift

```diff
@@ -258,8 +258,7 @@ extension Package {
                 AND b.status = 'ok'
                 GROUP BY b.platform
                 HAVING count(*) > 0
-            ),
-            updated_at = NOW()
+            )
```

Comment:
We're technically updating the package record but what we're really doing is aggregating up build data. So it's not the package per se that's changing.

A cleaner solution would perhaps be to move this aggregate field `platform_compatibility` to a separate table since we're using `packages` as a convenient place to stick it.

Maybe `packages_aggregates`. It could hold `platform_compatibility` and `score_details`. I'll add an issue to track.